docs/source/getting_started/dataset_creation/dataset_creation_gcloud_bucket.rst

```diff
@@ -1,6 +1,164 @@
+    :align: center
+    :alt: Google Cloud Service Account
+    :width: 60%
+
+- Click on `Done` to create the service account.
+- You can change the roles of the service account later in the
+  `IAM <https://console.cloud.google.com/iam-admin/iam>`_.
+
+6. Navigate to `APIs -> Credentials <https://console.cloud.google.com/apis/credentials>`_
+   again if you are not already there.
+
+- Find the just created user in the list of all service accounts.
+- Click on the user and navigate to the `keys` tab.
+- Click on `Add key` and create a new private key in JSON Format.
+  It will download the corresponding key file.
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_service_account_key_creation.jpg
+    :align: center
+    :alt: Google Cloud Service Account Key Creation
+    :width: 60%
+
+
+
+
+Configuring a Lightly dataset to access the Google Cloud Storage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create and configure a dataset
+
+1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly
+2. Edit your dataset and select `Google Cloud Storage` as your datasource
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_create_dataset.jpg
+    :align: center
+    :alt: Configure google cloud bucket datasource in Lightly Webapp
+    :width: 60%
+
+
+3. As the resource path, enter the full URI to your resource eg. `gs://lightly-datalake/projects/wild-animals`
+4. Enter the Google Project ID you wrote down in the first step.
+5. Click on `Select Credentials File` to add the key file you downloaded in the previous step.
+6. The thumbnail suffix allows you to configure
+
+- where your thumbnails are stored when you already have generated thumbnails in your S3 bucket
+- where your thumbnails will be stored when you want Lightly to create thumbnails for you.
+  For this to work, the user policy you created must be granted write permissions.
+- when the thumbnail suffix is not defined/empty,
+  the Lightly Webapp will load the full image even when requesting the thumbnail.
+
+
+6. Press save and ensure that at least the lights for List and Read turn green.
+If you added permissions for writing, this lights should also turn green.
+
+7. Now you should be on the dataset creation page again.
+
+Create the dataset and upload embeddings and metadata.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For creating the dataset and uploading embeddings and metadta to it, you need
+the :ref:`lightly-command-line-tool`. Furthermore, you need to have your data locally.
+This can be done easiest by using the `gsutil tool <https://cloud.google.com/storage/docs/gsutil>`_
+and its `rsync command <https://cloud.google.com/storage/docs/gsutil/commands/rsync>`_:
+
+.. code::
+
+    gsutil -m rsync -r /local/projects/wild-animals gs://datalake-lightly/projects/wild-animals
+
+
+Use `lightly-magic` and `lightly-upload` with the following considerations:
+
+- use `input_dir=/local/projects/wild-animals`
```

Comment:
unclear why this is part of the considerations?

Reply:
it sounds a little strange like this, maybe just mention that the input directory should be the root directory of your dataset?

Reply:
I used the same input_dir as in the gustil command above.